types/hls-parser/index.d.ts

```diff
@@ -94,6 +109,10 @@ export namespace types {
                 segments?: readonly Segment[];
                 prefetchSegments?: readonly PrefetchSegment[];
                 source?: string;
+                lowLatencyCompatibility?: LowLatencyCompatibility;
+                partTargetDuration?: number;
+                renditionReports?: RenditionReport[];
```

Comment:
Should we declare all arrays as `readonly` like all other arrays here?

Reply:
can't hurt for consistency I guess, I don't believe these are actually read-only though

Reply:
Usually they are not. But for consistency and prevent (accidental) mutation `readonly` would be nice 🙂